entry/src/entry.rs

```diff
@@ -111,15 +111,22 @@ pub struct Api<'a> {
 /// field points to Transactions that took place shortly before `hash` was generated.
 ///
 /// If you multiply `num_hashes` by the amount of time it takes to generate a new hash, you
-/// get a duration estimate since the last Entry. Since processing power increases
+/// get a duration estimate since the last `Entry`. Since processing power increases
 /// over time, one should expect the duration `num_hashes` represents to decrease proportionally.
 /// An upper bound on Duration can be estimated by assuming each hash was generated by the
 /// world's fastest processor at the time the entry was recorded. Or said another way, it
 /// is physically not possible for a shorter duration to have occurred if one assumes the
 /// hash was computed by the world's fastest processor at that time. The hash chain is both
 /// a Verifiable Delay Function (VDF) and a Proof of Work (not to be confused with Proof of
 /// Work consensus!)
-
+///
+/// `Entry` contains `transactions` that are executable in parallel. So an `Entry` is a unit of
+/// parallelism at the chain level. All transactions in the `transactions` field have to follow the
```

Comment:
Most of my prior experience is with Ethereum.  And it was also at the application level.
Still, I think, it is relatively unique to Solana, that recorded transactions are required to be executable in parallel.  In my mind, it seems to be an enforcement of the parallelism in the protocol itself.  It assumes that validators are not single threaded.
Ethereum VM, for example, is defined as a single thread of execution.

This is pretty interesting and, I think, deserves a sentence :)
Not sure why you think it is confusing and/or ungrounded.  Maybe I'm not saying it right, as English, is not my first language.

Not a big deal, so I've removed the sentence.
Though, if you'll find it educational, it would be interesting to hear why you do not see it this way.

Reply:
I guess it's that I don't really understand what a "unit of parallelism" is or what "the chain level" is. These aren't terms that are used or defined elsewhere in our code base (which is what I meant by ungrounded). Certainly this could just be my ignorance, if those terms are common elsewhere...?
I do get the distinction you're trying to make between Solana and single-thread-of-execution blockchains. But I land where Steve did, that the first sentence says what needs to be said.